hypertrace-core-graphql-platform/build.gradle.kts

```diff
@@ -23,18 +23,21 @@ dependencies {
     api("com.typesafe:config:1.4.1")
     api("com.google.guava:guava:30.1-jre")
     api("com.graphql-java-kickstart:graphql-java-servlet:10.1.0")
-    api("io.grpc:grpc-api:1.40.0")
-    api("io.grpc:grpc-core:1.40.0")
-    api("io.grpc:grpc-stub:1.40.0")
-    api("io.grpc:grpc-context:1.40.0")
+    api("io.grpc:grpc-api:1.42.0")
+    api("io.grpc:grpc-core:1.42.0")
+    api("io.grpc:grpc-stub:1.42.0")
+    api("io.grpc:grpc-context:1.42.0")
     api("com.fasterxml.jackson.core:jackson-databind:2.12.3")
     api("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.12.3")
     api("com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.12.3")
     api("org.apache.commons:commons-text:1.9")
     api("io.opentelemetry:opentelemetry-proto:1.1.0-alpha")
+    api("com.google.code.gson:gson:2.8.9") {
```

Comment:
This shouldn't be needed - are we still seeing issues?

Reply:
Let me remove and check

Reply:
Removed

Reply:
This is needed else snyk build fails

Reply:
yeah, just looked why - it's the intersection of a couple reasons:
- GQL runs snyk on all configs, not just runtime classpath. That's due to a bug in snyk where filtering classpath requires a match in every project. The runtime classpath correctly resolves without this.
- The compile classpath doesn't, because the grpc utils are encapsulated in graphql - so they don't show up on the compile classpath of the modues, by design. gson however does, because certain modules use protobuf-java-util directly for logging.

tl;dr - makes sense to leave it